exporter/datadogexporter/metrics_translator.go

```diff
@@ -140,6 +147,15 @@ func MapMetrics(logger *zap.Logger, cfg config.MetricsConfig, md pdata.Metrics)
 	rms := md.ResourceMetrics()
 	for i := 0; i < rms.Len(); i++ {
 		rm := rms.At(i)
+
+		var attributeTags []string
```

Comment:
Note for maintainers: we will make a future PR to increase unit test coverage on the map metrics function; it was hard to test a while back when everything used the OpenCensus format.